res/css/structures/_UserMenu.scss

```diff
@@ -128,7 +128,7 @@ limitations under the License.
 }
 
 .mx_UserMenu_contextMenu {
-    width: 247px;
+    width: 258px;
```

Comment:
what happens if a name given is too long? Does it ellipsis or overflow onto a new line?

Reply:
Ellipsis

Reply:
Note that this may happen if a user raises their font size too, where then it'll just say `Upgrade to Element Ho...`